### ISSUE TITLE
Entity: userdefined predefined type returns an error

### ISSUE THREAD

**Issue by giuseppeverduciALMA** [buildingSMART/IDS-Audit-tool#13]
IDS-Audit-tool reports this error:

**Error 103: Invalid value `"X"` to match `"PredefinedTypes"` (no valid values exist) in the context of Ifc2x3, Ifc4 on "simpleValue at line 22, position 26**

with this ids:
```
<ids xmlns="http://standards.buildingsmart.org/IDS"
    xmlns:xs="http://www.w3.org/2001/XMLSchema"
    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://standards.buildingsmart.org/IDS http://standards.buildingsmart.org/IDS/ids_09.xsd">
    <info>
        <title>Inherited predefined types should pass</title>
    </info>
    <specifications>
        <specification name="Inherited predefined types should pass" ifcVersion="IFC2X3 IFC4" minOccurs="1" maxOccurs="unbounded">
            <applicability>
                <entity>
                    <name>
                        <simpleValue>IFCWALL</simpleValue>
                    </name>
                </entity>
            </applicability>
            <requirements>
                <entity>
                    <name>
                        <simpleValue>IFCWALL</simpleValue>
                    </name>
                    <predefinedType>
                        <simpleValue>X</simpleValue>
                    </predefinedType>
                </entity>
            </requirements>
        </specification>
    </specifications>
</ids>
```
IDS test case: [[PASS] Inherited predefined types should pass](https://github.com/buildingSMART/IDS/blob/master/Documentation/testcases-entity.md#pass-inherited-predefined-types-should-pass)
The IDS element "predefinedType" contains either a predefinedType (defined by ifc) or an objectType ("X" in this case when USERDEFINED).

**Comment by CBenghi** [comment]
That's a correct observation, when `USERDEFINED` exists in the enumeration I should skip the check.

**Comment by CBenghi** [comment]
Solved in a previous commit 27223f7607e9298e1300d621d12026bf608f0bb7
